### PR TITLE
[r] Release 0.1.19

### PR DESCRIPTION
**Pull request by aaronwolen**
Updates package version number to 0.1.19 in preparation for release. 

Other changes:

- Updated `setup-r` GitHub Action to v2

Note: CI test failures are due to a core issue that will be fixed in 2.13.1.

### CODE REVIEW

**Comment by eddelbuettel** [comment]
Please change https://github.com/single-cell-data/TileDB-SOMA/blob/9bf705a08929ac0a310c6a61e4154d028c049d0d/.github/workflows/pkgdown.yaml#L19 to v3

File: apis/r/NEWS.md [hunk 1]
```diff
@@ -1,11 +1,22 @@
...
+
+# tiledbsoma 0.1.18
 
 ## Features
 
 - The `AnnotationMatrix`'s `to_matrix()` method now supports batched reads via the `batch_mode` argument. This functionality can also be leveraged from `SOMA`'s  `get_seurat_dimreductions_list()` and `get_seurat_dimreduction()` methods.
 
+## Changes
+
+- Increased capacity of `AssayMatrix` arrays from 1,000 to 100,000 to improve remote-read performance (#543)
```
Comment (johnkerl) [inline apis/r/NEWS.md:15]: This was a move from 100,000 down to 1,000 in order to increase read-locality for point queries

**Review by johnkerl — APPROVED** [review]
With one change requested 🚢

File: apis/r/NEWS.md (hunk 1 above)
Comment (aaronwolen) [reply apis/r/NEWS.md:15]: Good catch!

**Review by eddelbuettel — COMMENTED** [review]
Thanks for updating the checkout actions too

**Comment by aaronwolen** [comment]
Thanks for spotting that @eddelbuettel!

**Pull request merged**